pkg/services/contexthandler/contexthandler.go

```diff
@@ -306,20 +306,25 @@ func (h *ContextHandler) initContextWithAPIKey(reqContext *models.ReqContext) bo
 	*reqContext.Req = *reqContext.Req.WithContext(ctx)
 
 	var (
-		apikey *apikey.APIKey
+		aKey   *apikey.APIKey
```

Comment:
```suggestion
		apiKey   *apikey.APIKey
```

Reply:
Found that it was too similar to the package name.

Let me marinate